Editor/Resources/EditorWindow/Pages/LandingPage.uxml

```diff
@@ -46,17 +46,26 @@
                 <ui:VisualElement class="separator">
                     <ui:Image name="CaseStudy1" class="case-study__image"/>
                     <ui:Label name="LandingPageCaseStudy1Description" text="Lorem ipsum dolor sit amet, \n consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."/>
-                    <ui:Label name="LearnMore" text="Learn More" class="external-link"/>
+                    <ui:VisualElement name="CaseStudy1LearnMore" class="external-link">
+                        <ui:Label text="Learn More"/>
+                        <ui:Image name="ExternalLinkIcon" class="icon--small"/>
+                    </ui:VisualElement>
                 </ui:VisualElement>
                 <ui:VisualElement class="separator">
                     <ui:Image name="CaseStudy2" class="case-study__image"/>
                     <ui:Label name="LandingPageCaseStudy2Description" text="Lorem ipsum dolor sit amet, \n consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."/>
-                    <ui:Label name="LearnMore" text="Learn More" class="external-link"/>
+                    <ui:VisualElement name="CaseStudy2LearnMore" class="external-link">
+                        <ui:Label text="Learn More"/>
```

Comment:
These need to come from the TextProvider

Reply:
Solved